extensions/cache/src/main/resources/benchmark-redis.xml

```diff
@@ -0,0 +1,69 @@
+<benchmark xmlns="urn:radargun:benchmark:3.0">
+
+   <master bindAddress="${master.address:127.0.0.1}" port="${master.port:2103}"/>
+
+   <clusters>
+      <cluster size="8">
```

Comment:
Do we need such "big" setup? Example configuration should consume as few resources as possible.
Have you ran this locally using the `bin/tests.sh` script?
You could also add a comment suggesting that you run one redis per core even in real benchmark.